source/Playnite/Database/GameDatabase.cs

```diff
@@ -1059,34 +1059,60 @@ public List<Game> ImportGames(LibraryPlugin library, bool forcePlayTimeSync, ILi
                     }
                     else
                     {
-                        existingGame.IsInstalled = newGame.IsInstalled;
-                        existingGame.InstallDirectory = newGame.InstallDirectory;
+                        var existingGameUpdated = false;
+                        if (existingGame.IsInstalled != newGame.IsInstalled)
+                        {
+                            existingGame.IsInstalled = newGame.IsInstalled;
+                            existingGameUpdated = true;
+                        }
+                        
+                        if (existingGame.InstallDirectory.Equals(newGame.InstallDirectory, StringComparison.OrdinalIgnoreCase) == false)

```

Comment:
This will throw if `InstallDirectory` is null.

Reply:
Fixed by using `string.Equals` method instead.